src/modules/dashboard/lists/BorrowedPositionsList/BorrowedPositionsList.tsx

```diff
@@ -72,13 +76,85 @@ export const BorrowedPositionsList = () => {
         .div(maxBorrowAmount)
         .toFixed();
 
+  if (sortDesc) {
+    if (sortName === 'symbol') {
+      borrowPositions.sort((a, b) =>
+        a.reserve.symbol.toUpperCase() < b.reserve.symbol.toUpperCase() ? -1 : 1
+      );
+    } else if (sortName === 'debt') {
+      borrowPositions.sort((a, b) => Number(a.variableBorrows) - Number(b.variableBorrows));
+    } else {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      borrowPositions.sort((a, b) =>
+        a.borrowRateMode === 'Variable'
+          ? Number(a.reserve.variableBorrowAPY) - Number(b.reserve.variableBorrowAPY)
+          : Number(a.reserve.stableBorrowAPY) - Number(b.reserve.stableBorrowAPY)
+      );
+    }
+  } else {
+    if (sortName === 'symbol') {
+      borrowPositions.sort((a, b) =>
+        b.reserve.symbol.toUpperCase() < a.reserve.symbol.toUpperCase() ? -1 : 1
+      );
+    } else if (sortName === 'debt') {
+      borrowPositions.sort((a, b) => Number(b.variableBorrows) - Number(a.variableBorrows));
+    } else {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      // borrowPositions.sort((a, b) => b[sortName] - a[sortName]);
+      borrowPositions.sort((a, b) =>
+        a.borrowRateMode === 'Variable'
+          ? Number(b.reserve.variableBorrowAPY) - Number(a.reserve.variableBorrowAPY)
+          : Number(b.reserve.stableBorrowAPY) - Number(a.reserve.stableBorrowAPY)
+      );
+    }
+  }
```

Comment:
Again here too, I think because of just the nuances in this logic it may be useful to pepper it with comments for each branch of logic and/or one overall comment on line 78 explaining at a high level.

It's too bad we can't DRY these statements up, as they're all slightly different